build.sbt

```diff
@@ -11,7 +11,7 @@ scalaVersion := "2.10.4"
 scalacOptions += "-feature"
 
 libraryDependencies ++= Seq(
-  "org.webjars" % "rjs" % "2.1.15"
+  "org.webjars" % "requirejs-node" % "2.1.17"
```

Comment:
I'm not really sure why to prefer `rjs` or `requirejs-node`. I wonder if just upgrading to `"org.webjars" % "rjs" % "2.1.19"` might be safer since it seems like a smaller change...